spec/controllers/hyrax/paper_or_reports_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Generated via
+#  `rails generate hyrax:work PaperOrReport`
+require 'rails_helper'
+
+RSpec.describe Hyrax::PaperOrReportsController do
+  let(:user) { FactoryBot.create(:user) }
+  let(:work) { FactoryBot.create(:paper_or_report_with_one_file, user: user) }
+  let(:file_set) { work.ordered_members.to_a.first }
+
+  before do
+    Hydra::Works::AddFileToFileSet.call(file_set,
+                                        fixture_file('images/world.png'),
+                                        :original_file)
+  end
+
+  describe "#presenter" do
+    subject { controller.send :presenter }
+
+    let(:solr_document) { SolrDocument.new(FactoryBot.create(:generic_work).to_solr) }
```

Comment:
```suggestion
    let(:solr_document) { SolrDocument.new(FactoryBot.create(:work).to_solr) }
```

Reply:
If we want to use the `PaperOrReport` created on line 9, I think we actually want this: 

```suggestion
    let(:solr_document) { SolrDocument.new(work).to_solr) }
```

Reply:
you're right!

Reply:
good catch. that's what happens when you copy paste